web/modules/custom/dbcdk_community/src/Profile/HasQuarantinesTrait.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * @file
+ * Definition of HasQuarantinesTrait.
+ */
+
+namespace Drupal\dbcdk_community\Profile;
+
+/**
+ * This trait is useful when extending model classes related to quarantines.
+ */
+trait HasQuarantinesTrait {
+
+  /**
+   * The related quarantines.
+   *
+   * @var \DBCDK\CommunityServices\Model\Quarantine[] $quarantines
+   */
+  protected $quarantines = [];
+
+  /**
+   * Get quarantines related to this object.
+   *
+   * @return \DBCDK\CommunityServices\Model\Quarantine[]
+   *   Related quarantines.
+   */
+  public function getQuarantines() {
+    return $this->quarantines;
+  }
+
+  /**
+   * Set quarantines related to this object.
+   *
+   * @param |DBCDK\CommunityServices\Model\Quarantine[] $quarantines
```

Comment:
Scrutinizer caught this: the namespace should start with a backslash, not a pipe